game/src/Game/GameSession/billboard.gdshader

```diff
@@ -0,0 +1,41 @@
+shader_type spatial;
+//depth_test_disabled along with the sorting offset of 10
+//ensures this will be drawn in front of other objects in 
+//3d space.
+render_mode unshaded, depth_test_disabled;
+
+
+//vic2 only ever loads a max of 12 BillboardType
+//Of these, only 4 are actually used
+uniform sampler2D billboards[12] : source_color;
+uniform uint numframes[12];
+uniform float sizes[12];
+
+uniform float height_add_factor = 0.3;
```

Comment:
I don't think this is needed, when I remove it the icons appear closer to their Vic2 positions, whereas with it they're all a bit further to the north